src/worker/move-contact-worker.ts

```diff
@@ -0,0 +1,160 @@
+import axios from 'axios';
+import { spawn } from 'child_process'; 
+import { Worker, Job } from 'bullmq';
+
+import Auth from '../lib/authentication';
+import { queuePrivateKey, redisConnection } from '../shared/queue-config';
+import { queueManager } from '../shared/queues';
+
+export interface MoveContactData {
+  parentId: string;
+  contactId: string;
+  sessionToken: string;
+  instanceUrl: string;
+}
+
+export type JobResult = { success: boolean; message: string };
+
+export class MoveContactWorker {
+  private readonly DELAY_IN_MILLIS = 360_000;   // 1 hour
+  private readonly MAX_TIMEOUT = 180_000;       // 30 minutes timeout
+  private readonly MAX_CONCURRENCY = 1;         // Limit concurrency to 1 job at a time
+  private readonly MAX_SENTINEL_BACKLOG = 1000; // ensure we don't take down the server
```

Comment:
roughly the number we have for `MAX_SENTINEL_BACKLOG` should be approximately how much of the sentinel backlog we expected to be processed within the `DELAY_IN_MILLIS`

[Here](https://github.com/moh-kenya/config-echis-2.0/issues/2174#issuecomment-2125502935) is a random single datapoint we can use. Looks like about 20,000 docs were processed in 3 hours.  So maybe let's do ~7000?